### DISCUSSION TITLE
Release 10.9?

### DISCUSSION THREAD

**Discussion by citronalco** [jellyfin/jellyfin#10813] [General] [answered]
10.8.0 was released in June 2022. There've been >2500 Commits in master since release 10.8. Many of them sound very promising, and it's a pity most people are missing them - except they install the nightly builds.

So: Will there be a release 10.9 in the near future?

**Comment by crobibero** [comment, marked as answer]
https://jellyfin.org/docs/general/faq/#when-will-release-x-happen-when-will-feature-x-be-available

**Comment by citronalco** [comment]
I'll take this as a "no, nothing planned yet".

Thanks for your work, Jellyfin is great!

**Comment by flashlab** [comment]
@citronalco May I ask where the nightly builds is?

**Comment by citronalco** [comment]
@flashlab I meant the unstable builds. They seem to be updated every few days (so not really nightly).
